parl/remote/client.py

```diff
@@ -274,16 +275,31 @@ def _check_and_monitor_job(self, job_heartbeat_address,
 
         # a thread for sending heartbeat signals to job
         thread = threading.Thread(
-            target=self._create_job_monitor, args=(job_heartbeat_socket, ))
+            target=self._create_job_monitor,
+            args=(job_heartbeat_socket, proxy_wrapper_nowait_object))
         thread.setDaemon(True)
         thread.start()
         return True
 
-    def _create_job_monitor(self, job_heartbeat_socket):
+    def _create_job_monitor(self, job_heartbeat_socket,
+                            proxy_wrapper_nowait_object):
         """Send heartbeat signals to check target's status"""
+        copy_proxy_wrapper_nowait_object = proxy_wrapper_nowait_object
+
+        # @TODO(zenghsh3): hardcoded
+        actor_deleted_refcount = 7
```

Comment:
Please rewrite this part of code to separate this part of logic.

Reply:
done